include/updater/updater_config.hpp

```diff
@@ -0,0 +1,81 @@
+/*
+
+Copyright (c) 2016, Project OSRM contributors
```

Comment:
Just curious about license boilerplate, would be enough just to reference [LICENSE.txt](http://softwareengineering.stackexchange.com/a/19653)?